src/rpc/rpcquorums.cpp

```diff
@@ -61,7 +165,11 @@ UniValue quorum(const JSONRPCRequest& request)
 
     std::string command = request.params[0].get_str();
 
-    if (command == "dkgstatus") {
+    if (command == "list") {
```

Comment:
Should probably add `quorum_help()` with the list of available commands, smth similar to `protx_help()`, and also unify the logic flow here with the one in `protx()` (and other "complex" rpc commands).

Reply:
done